app/workers/organizations/bust_cache_worker.rb

```diff
@@ -3,9 +3,9 @@ class BustCacheWorker < BustCacheBaseWorker
     def perform(organization_id, slug)
       organization = Organization.find_by(id: organization_id)
 
-      return unless organization
+      return unless organization && slug
```

Comment:
Minor optimization suggestion: Should we bother running a `find_by` on `Organization` if `slug` isn't present?

Reply:
Good idea - updated!